src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
Move this file to a folder in the root directory and name the folder `__tests__` . Tests usually live in their own directory, far from the runtime code